openwisp-config/files/openwisp.agent

```diff
@@ -718,7 +741,7 @@ update_configuration() {
 	REMOTE_CHECKSUM=$(tail -n 1 $CONFIGURATION_CHECKSUM 2>/dev/null)
 
 	if [ "$LOCAL_CHECKSUM" != "$REMOTE_CHECKSUM" ]; then
-		logger -s "Configuration checksum mismatch! Local: $LOCAL_CHECKSUM, Remote: $REMOTE_CHECKSUM"  \
+		logger -s "Configuration checksum mismatch! Local: $LOCAL_CHECKSUM, Remote: $REMOTE_CHECKSUM" \
```

Comment:
I am going to fix this in master now.

Reply:
Done: https://github.com/openwisp/openwisp-config/commit/41a1590fe4688ff2a85fb6f5f7a25f4e23d97f48